dask/dataframe/io/parquet.py

```diff
@@ -0,0 +1,170 @@
+import os
+import struct
+
+from toolz import first
+
+import dask
+from dask import delayed, compute
+from dask.bytes.core import open_files, OpenFileCreator
+import dask.dataframe as dd
+from dask import delayed
+
+try:
+    import fastparquet
+    from fastparquet import parquet_thrift
+    default_encoding = parquet_thrift.Encoding.PLAIN
+except:
+    fastparquet = False
+    default_encoding = None
```

Comment:
That's fine with me, but it will have implications on how we reference these functions with dd.read_parquet, DataFrame.to_parquet.